sandbox/create_admin.py

```diff
@@ -0,0 +1,32 @@
+import os
+from redisConnections.redis_users_connection import RedisUsersConnection
+
+
+if __name__ == '__main__':
+    users = RedisUsersConnection()
+
+    admin_id = users.id_by_username('admin')
+    if admin_id != b'':
+        print("Admin user already exists.")
+        exit(0)
+
+    # Getting confd-credentials from yangcatalog config
+    confd_creds = None
+    with open(os.environ['YANGCATALOG_CONFIG_PATH'], 'r') as f:
```

Comment:
Please use `create_config(config_path)` method call - you can find it in almost each script, just check how it works. Then you are able to get value from config by key.